package.json

```diff
@@ -5,6 +5,8 @@
   "description": "",
   "scripts": {
     "buildForms": "node ./src/buildForms.js",
+    "lint": "npm run lint:ttl",
+    "lint:ttl": "find ./dist -type f -name \"*.ttl\" | xargs npx turtle-validator",
```

Comment:
This will not work on Windows, but I think that's fine for now. We can probably create a `./src/validateForms.js` in the future that does the same thing but in node code, which would work under Windows as well.

Reply:
I've tested it on both linux and windows environments with the VSC editor using git bash and it does work.

That's a nice addition to have :+1: